packages/rocketchat-slashcommands-leave/server/leave.js

```diff
@@ -0,0 +1,31 @@
+
+/*
+* Leave is a named function that will replace /leave commands
+* @param {Object} message - The message object
+*/
+function Leave(command, params, item) {
+	if (command !== 'leave' && command !== 'part') {
+		return;
+	}
+	try {
+		Meteor.call('leaveRoom', item.rid);
+	} catch ({error}) {
+		RocketChat.Notifications.notifyUser(Meteor.userId(), 'message', {
+			_id: Random.id(),
+			rid: item.rid,
+			ts: new Date,
+			msg: TAPi18n.__(error, null, Meteor.user().language)
+		});
+	}
+}
+if (Meteor.isClient) {
```

Comment:
I think this if will not be true here, never. you should split the code in client and server instead duplicate.